cmd/earthly/root_cmds.go

```diff
@@ -311,6 +311,30 @@ Set up a whole custom git repository for a server called example.com, using a si
 				},
 			},
 		},
+		{
+			Name:        "web-ui",
```

Comment:
Shouldn't this have been under `account`? `earthly account web-ui`?

Reply:
There was a minor remark in the internal Slack thread where simply `earthly web-ui` was proposed.
I opted for that, but open to change it.

Reply:
I'm kinda inclined towards the more structured option, under `account`, but it's not a strong opinion.

Separately, thoughts on using just `web`? Like `earthly web` or `earthly account web`? Feels cleaner.

Reply:
Funny enough, I was having an internal debate when I was writing this and just `earthly web` crossed my mind.
I'm open to it.

Reply:
Votes are in - `web` it is! :)